aws/offsite/prune-aurora-backups/README.md

```diff
@@ -27,22 +27,38 @@ sam local invoke PruneAuroraBackupsFunction --event event.json --profile <aws cl
 
 ## Deployment
 
+1. **Configure AWS Client** - ensure local AWS API calls authenticate with credentials that can deploy to the offsite AWS account.
+    1. **Create Offsite Profile** - Add an entry to `.aws/config` named `codeorg-offsite` such as
+      ```
+        [profile codeorg-offsite]
+        region = my-region-2
+      ```
+   2. **Configure Credentials** - Add an entry to `.aws/credentials` with API keys provisioned with appropriate Role/Policies
+     ```
+       [codeorg-offsite]
+        aws_access_key_id = MYKEYID
+        aws_secret_access_key = mysecretkey
+     ```
+   3. **Switch to Offsite Profile** - Configure the AWS client to default to the offsite profile
+    ``` bash
+      export AWS_PROFILE=codeorg-offsite
+    ```
```

Comment:
This is tightly coupled to the implementation details of aws-google and should probably be kept simple (the first line).

Reply:
Same as with the other deployment steps, we can't use our `aws-google` gem so credentials for the offsite account have to be configured manually.